src/components/dropdown/dropdown.component.js

```diff
@@ -41,7 +41,9 @@ const ScDropdownComponent = {
     bindings: {
         closeOnSelect: '<',
         onOpen: '&',
-        side: '@'
+        side: '@',
+        visible: '=',
```

Comment:
This is an input for initialisation, it should be named something like visibleOnInit
You should not use double binding